npm/nwpolicy.go

```diff
@@ -189,3 +198,35 @@ func (npMgr *NetworkPolicyManager) DeleteNetworkPolicy(npObj *networkingv1.Netwo
 
 	return nil
 }
+
+func createCidrsRule(ingressOrEgress string, policyName string, ipsetEntries [][]string, ipsMgr *ipsm.IpsetManager) {
+	spec := append([]string{util.IpsetNetHashFlag, util.IpsetMaxelemName, util.IpsetMaxelemNum})
+	for i, ipCidrSet := range ipsetEntries {
+		if ipCidrSet == nil || len(ipCidrSet) == 0 {
+			continue
+		}
+		setName := policyName + "-cidr" + strconv.Itoa(i) + ingressOrEgress
+		log.Printf("Creating set: %v, hashedSet: %v", setName, util.GetHashedName(setName))
+		if err := ipsMgr.CreateSet(setName, spec); err != nil {
+			log.Printf("Error creating ipset %s", ipCidrSet)
+		}
+		for _, ipCidrEntry := range util.DropEmptyFields(ipCidrSet) {
+			if err := ipsMgr.AddToSet(setName, ipCidrEntry, util.IpsetNetHashFlag); err != nil {
+				log.Printf("Error adding ip cidrs %s into ipset %s", ipCidrEntry, ipCidrSet)
+			}
+		}
+	}
+}
+
+func removeCidrsRule(ingressOrEgress string, policyName string, ipsetEntries [][]string, ipsMgr *ipsm.IpsetManager) {
+	for i, ipCidrSet := range ipsetEntries {
+		if ipCidrSet == nil || len(ipCidrSet) == 0 {
```

Comment:
What I intend to do here is, delete ipset npm created when translate cidrs rule. Length of the set includes cidrs belong to this rule.
It works as intended. I test it by updating same network policy rules. It will remove and update the target ipset. 
![image](https://user-images.githubusercontent.com/16043262/84828785-768ef380-afdb-11ea-8a9a-aa98b317be58.png)